system_meta/management/commands/manage_product.py

```diff
@@ -0,0 +1,310 @@
+"""
+Manages products within the app.
+
+Return codes:
+0: Success
+1: Product does not exist
+2: Product already exists
+
+Ignoring A003 because "help" is valid for argparse.
+"""
+# ruff: noqa: A003, PLR0913, FBT002
+import argparse
+
+from django.core.management import BaseCommand, CommandError
+from mitol.common.utils.datetime import now_in_utc
+from prettytable import PrettyTable
+
+from system_meta.models import IntegratedSystem, Product
+
+
+class Command(BaseCommand):
+    """
+    Manages products within the app.
+    """
+
+    help = "Manages products within the app."
+
+    def add_arguments(self, parser):
+        """Parse arguments provided to the command."""
+
+        sku_system_args = argparse.ArgumentParser(add_help=False)
+        sku_system_args.add_argument(
+            "--sku",
+            type=str,
+            required=True,
+            help="The product's SKU.",
+            metavar="sku",
+        )
+        sku_system_args.add_argument(
+            "--system",
+            "-s",
+            type=str,
+            required=True,
+            help="The system to add the product to.",
+            metavar="system",
+        )
+
+        crud_args = argparse.ArgumentParser(add_help=False)
+        crud_args.add_argument(
+            "--price",
+            type=float,
+            required=True,
```

Comment:
Looks like this probably shouldn't be required since it's shared with the update command, which means we should either:

- Make it optional and just do a simple check in `_add_product` for required columns and raise an error otherwise.
- Define these args separately for both `add` and `update`.

I'm fine with either approach.

Reply:
Got this in https://github.com/mitodl/unified-ecommerce/pull/31/commits/e8f7abfa411e3c6902995ffa9f70814dddfcd020 (went with option 2)